terms.html

```diff
@@ -9,7 +9,7 @@
       Terms of Service
```

Comment:
@mtrang1263 - Is there any way we can comment this page out (or something similar) for launch? Nicole has said that the terms of service not being approved is no longer a blocker, but I'd prefer not to launch with a blank page.

Reply:
This was achieved by adding the terms page to our exclusion section. Created this ticket to re-enable when ready: https://jira.cms.gov/browse/BCDA-4024